tests/unit-core/test-api-strings.c

```diff
@@ -159,6 +159,22 @@ main (void)
     jerry_value_free (test_str);
   }
 
+  /* Test jerry_string_substr */
+  {
+    jerry_value_t test_str = jerry_string_sz ("Hello World!");
```

Comment:
Prefer `JERRY_ZSTR_ARG`

Reply:
Its just a test code. It does not make any difference.